charts/performancetesting/templates/cron.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Chart.Name }}-cron
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: uosjarchart
```

Comment:
can we change `uosjarchart` to `performancetest`

Reply:
Done! Thanks for the comment